examples/platform/mbed/capsense/capsense.cpp

```diff
@@ -0,0 +1,220 @@
+
+#include "capsense.h"
+
+using namespace std::literals::chrono_literals;
+
+#define SLIDER_NUM_TOUCH (1u)
+#define LED_OFF (1u)
+#define LED_ON (0u)
+
+#define CAPSENSE_SCAN_PERIOD_MS (20ms)
+#define EZI2C_INTERRUPT_PRIORITY (3u)
+
+#define TUNER_ENABLE (1u)
```

Comment:
Consider using `constexpr` instead of `#define`

Reply:
This causing compiler error